Readme.md

```diff
@@ -5,40 +5,76 @@
 
-This example shows how to display custom text in data cells. In this example, the [GridControl](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.GridControl) adds the `(SALE)` string to the **Product Name** if a value in the **Discount** column is greater than 20.
+In this example, the [`GridControl`](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.GridControl) changes cell display text when a specific condition is met. 
 
-![](https://docs.devexpress.com/WPF/images/GridControl_CustomColumnDisplayText.png?v=21.2)
+If the **Discount** value is greater than 20, the grid appends **(SALE)** to the **Product Name**.
 
-<!-- default file list -->
+![Custom Text in Cells Based on a Condition](./Images/custom-text-in-cells.jpg)
 
-## Files to Look At
+Use this technique when you need to:
 
-### Code Behind Technique
+- Add labels or status markers to cell values.
+- Update cell text dynamically while the original data remains unchanged.
+- Improve readability and highlight key values.
 
-- [MainWindow.xaml](./CS/DisplayCustomText_CodeBehind/MainWindow.xaml) ([MainWindow.xaml](./VB/DisplayCustomText_CodeBehind/MainWindow.xaml))
-- [MainWindow.xaml.cs](./CS/DisplayCustomText_CodeBehind/MainWindow.xaml.cs#L20-L25) ([MainWindow.xaml.vb](./VB/DisplayCustomText_CodeBehind/MainWindow.xaml.vb#L22-L29))
+## Implementation Details
 
-### MVVM Technique
+The example includes [Code-Behind](#code-behind) and [MVVM](#mvvm) techniques.  
 
-- [MainWindow.xaml](./CS/DisplayCustomText_MVVM/MainWindow.xaml) ([MainWindow.xaml](./VB/DisplayCustomText_MVVM/MainWindow.xaml))
-- [MainViewModel.cs](./CS/DisplayCustomText_MVVM/MainViewModel.cs#L32-L40) ([MainViewModel.vb](./VB/DisplayCustomText_MVVM/MainViewModel.vb#L76-L84))
+### Code-Behind
 
-<!-- default file list end -->
+Handle the [`CustomColumnDisplayText`](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.GridControl.CustomColumnDisplayText) event to replace or extend the text displayed in a cell based on custom conditions.
+
+```csharp
+void grid_CustomColumnDisplayText(object sender, CustomColumnDisplayTextEventArgs e) {
+    if (e.Column.FieldName == "ProductName") {
+        decimal discount = (decimal)e.GetCellValue("Discount");
+        if (discount > 20)
+            e.DisplayText += " (SALE)";
+    }
+}
+```
+
+### MVVM
+
+Bind the [CustomColumnDisplayTextCommand](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.GridControl.CustomColumnDisplayTextCommand) to a view model command to change the text displayed in a cell based on custom conditions.
+
+```csharp
+public void OnCustomColumnDisplayText(CustomColumnDisplayTextEventArgs e) {
+    if (e.Column.FieldName == "ProductName") {
+        decimal discount = (decimal)e.GetCellValue("Discount");
+        if (discount > 20)
+            e.DisplayText += " (SALE)";
+    }
+}
+```
+
+## Files to Review
+
+### Code-Behind
+
+* [MainWindow.xaml](./CS/DisplayCustomText_CodeBehind/MainWindow.xaml) ([VB](./VB/DisplayCustomText_CodeBehind/MainWindow.xaml))
```

Comment:
On this line (and line 59 below), are you missing the VB file name for link text?

Reply:
Ok, thanks, I will fix it